### PR TITLE
Added correct translation in Malayalam 

### PR DESCRIPTION
**Pull request by NishithP2004**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link]https://nishithp2004.github.io/gci18.fossasia.org/

- Fixes #<Issue Number>

## Description / Changes
Hi I have added the correct translation in Malayalam.

### After:

- - - - - - - - - - - -

### CODE REVIEW

**Comment by jogendra** [comment]
@NishithP2004 Sorry, updated the comment, Why did you deleted CNAME file? there shouldn't be any change in CNAME file.

**Comment by NishithP2004** [comment]
Yeah that’s because in the initial PR I was facing some issues in uploading it to the GitHub pages but there is no issue now.

**Comment by NishithP2004** [comment]
@jogendra

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-gci18.fossasia.org
```
Comment (abishekvashok) [inline CNAME]: Please don't delete this

**Comment by NishithP2004** [comment]
@abishekvashok Hi I have updated the CNAME.

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**